metabase/metabase
# Add explanatory toast and `Undo` when auto-applying filters

Epic: https://github.com/metabase/metabase/issues/35147

### Description



### How to verify

_**Adding a dashcard**_
1. Set up a dashboard with one or two dashcards and apply a filter to them
2. Add a dashcard.
3. Check the dashcard filter, you should see that it's connected to the parameter that was applied to the dashcards
4. See that there's an "Undo" banner. Clicking "Undo auto-connection" should prevent that card from subscribing to or setting auto-wired parameters for cards

_**Wiring existing dashcards**_

1. Set up a dashboard with two dashcards and wire them with the same parameter
2. Once the auto-wiring is set, see that there's an undo banner.
3. Clicking "Undo auto-connection" will disable the ability for that dashcard to set auto-wired parameters, or receive auto-wired parameters.
